### DISCUSSION TITLE
Is there a way to add the social icons to the about page.

### DISCUSSION THREAD

**Discussion by sroeschus** [HEIGE-PCloud/DoIt#791] [Q&A] [answered]
I was wondering if it is possible to add the social icons also to the about page (like in the home page). What I mean by the social icons are the icons like: linkedin, twitter, email....

**Comment by HEIGE-PCloud** [comment, marked as answer]
You can directly use icons from font awesome in the markdown file

https://hugodoit.pages.dev/theme-documentation-content/#fontawesome
